libs/common-services/src/main/java/uk/gov/di/ipv/core/library/gpg45/Gpg45Scores.java

```diff
@@ -59,20 +62,17 @@ public Gpg45Scores(
             int activity,
             int fraud,
             int verification) {
-        this(Arrays.asList(evidence1, evidence2, evidence3), activity, fraud, verification);
+        this.evidences = sortEvidence(Arrays.asList(evidence1, evidence2, evidence3));
+        this.activity = activity;
+        this.fraud = fraud;
+        this.verification = verification;
     }
 
     public Gpg45Scores(List<Evidence> evidence, int activity, int fraud, int verification) {
+        this.evidences = sortEvidence(evidence);
         this.activity = activity;
         this.fraud = fraud;
         this.verification = verification;
-        this.evidences =
-                evidence.stream()
-                        .sorted(
-                                Comparator.comparing(Evidence::getStrength)
-                                        .thenComparing(Evidence::getValidity)
-                                        .reversed())
-                        .toList();
```

Comment:
Sorry, I think you misunderstood my comment - if you make all the other constructors delegate to this one, then you only need to do the sorting once :)

e.g.:
```
    public Gpg45Scores(Evidence evidence, int activity, int fraud, int verification) {
       this(List.of(evidence), activity, fraud, verification);
    }

    public Gpg45Scores(
            Evidence evidence1, Evidence evidence2, int activity, int fraud, int verification) {
        this(List.of(evidence1, evidence2), activity, fraud, verification);
    }

   // etc.

    public Gpg45Scores(List<Evidence> evidence, int activity, int fraud, int verification) {
        this.evidences = sortEvidence(evidence);
        this.activity = activity;
        this.fraud = fraud;
        this.verification = verification;
    }
```
I think this is a lot neater anyway, and good practice as it gives you a single place to add new constructor logic (like the sorting...).

Reply:
Ah I see! I follow what you mean. I can make those changes